tm/tm.config.js

```diff
@@ -0,0 +1,2 @@
+const key = 'dijuQaQgNxRJLxYmi5ZlROnW7RG0c4kC';
+module.exports = key;
```

Comment:
@tedgreen29 This is a public repo. I don't think you want to check-in your key 😃